s3/pkg/meta/db/db.go

```diff
@@ -46,8 +46,6 @@ type DBAdapter interface {
 	//PutObjectMap(objMap *ObjMap, tx interface{}) error
 	//DeleteObjectMap(objMap *ObjMap, tx interface{}) error
 
-	//cluster
-	GetCluster(fsid, pool string) (cluster Cluster, err error)
 	//lc
 	/*
```

Comment:
please remove unused code

Reply:
@wisererik thanks for your review, and the unused code has been removed.